packages/tkg-clusterclass-aws/bundle/config/upstream/base.yaml

```
@@ -1027,7 +1027,9 @@ spec:
     spec:
       kubeadmConfigSpec:
         files: []
-        preKubeadmCommands: []
+        preKubeadmCommands:
+        # Ensure a non-empty preKubeadmCommands array is present to facilitate patching
```

In all other places we are using `#!` to avoid ytt validation errors. We should use the same here as well.
```suggestion
        #! Ensure a non-empty preKubeadmCommands array is present to facilitate patching
```